src/runtime/objmodel.cpp

```diff
@@ -3180,9 +3180,17 @@ void rearrangeArguments(ParamReceiveSpec paramspec, const ParamNames* param_name
     std::vector<Box*, StlCompatAllocator<Box*>> varargs;
     if (argspec.has_starargs) {
         assert(!rewrite_args);
-        Box* given_varargs = getArg(argspec.num_args + argspec.num_keywords, arg1, arg2, arg3, args);
-        for (Box* e : given_varargs->pyElements()) {
-            varargs.push_back(e);
+        Box* given_varargs = nullptr;
+        try {
+            given_varargs = getArg(argspec.num_args + argspec.num_keywords, arg1, arg2, arg3, args);
+            for (Box* e : given_varargs->pyElements()) {
+                varargs.push_back(e);
+            }
+        } catch (ExcInfo const& e) {
```

Comment:
ok, I didn't know that it is required because of that. Good to know. Will fix that